infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -14,7 +14,7 @@
   {
     "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.ExternalUserException",
     "id": 3,
-    "reason": "The requested item does not exist in out repository",
+    "reason": "Your request has caused an exception. Please consult the logs for more information.",
```

Comment:
@manstis This doesn't really make sense as customers don't have access to the logs as this is a managed service. I'd say let's go generic e.g "We were unable to process your request. Please contact Red Hat support if the error persists".

We can then extend Error reporting in the future to perhaps include a request id (or something) that makes it easier to speak with support and support to trace their request.

Reply:
It makes more sense than the old message; but I'll update to your proposal.